utils/perl5lib/Batch/BatchMaker.pm

```diff
@@ -278,47 +280,47 @@ sub getField()
 }
 
 #==============================================================================
-# Get the batch directives for this machine from config_batch.xml, 
+# Get the batch directives for this machine from config_batch.xml,
 #==============================================================================
 sub setBatchDirectives()
 {
     my $self = shift;
     my $batchparser = $self->getBatchConfigParser();
     my $configmachinesparser = $self->getConfigMachinesParser();
-    
-    # get the batch directive for this particular queueing system. 
 
-    my @batch_directive = $batchparser->findnodes("/config_batch/batch_system[\@type=\'$self->{'batch_system'}\']/batch_directive");
+    # get the batch directive for this particular queueing system.
+
+    my @batch_directive = $batchparser->findnodes("/config_batch/batch_system[\@type=\'$self->{'batchtype'}\']/batch_directive");
 
     if(!@batch_directive)
     {
-	die "Cannot find batch directive for the batch system type $self->{'batch_system'}";
+	die "Cannot find batch directive for the batch system type $self->{'batchtype'}";
```

Comment:
xpath to batch_directive value different in config than in case.   this is only a perl problem.